### DISCUSSION TITLE
Help - Can't get modules to run

### DISCUSSION THREAD

**Discussion by jpetrone84** [codeproject/CodeProject.AI-Server#229] [Q&A]
I've been trying to get CodePrjoject AI server up and running with a few modules but for whatever reason the modules are having trouble. The only one I was able to get started is Face Processing 1.12.3. License Plate Reader 3.3.3, will start and then immediately stop. Object Detection (Coral) 2.4.0 recognizes the Coral while starting and then immediately closes with "Unknown descriptor from device". Sound Classifier (TensorFlow) 1.4.0 will immediately stop too.  I've attached logs. Some say self test failed which obviously doesn't sound like a good thing  but I don't know why things are failing and nothing is obvious to me as to the reason why. I'm sure it's something stupid, that I'm not doing correctly because I just don't know what I'm doing. I appreciate any help. 

[License Plate Reader 3.3.3.txt](https://github.com/user-attachments/files/18115898/License.Plate.Reader.3.3.3.txt)
[Object Detection (Coral) 2.4.0.txt](https://github.com/user-attachments/files/18115899/Object.Detection.Coral.2.4.0.txt)
[server log.txt](https://github.com/user-attachments/files/18115900/server.log.txt)
[Sound Classifier (TensorFlow) 1.4.0.txt](https://github.com/user-attachments/files/18115901/Sound.Classifier.TensorFlow.1.4.0.txt)

**Comment by ChrisMaunder** [comment]
Can you please try uninstalling then reinstalling the ALPR moodule with verbosity set to 'loud'

**Reply by jpetrone84** [reply]
Thanks for helping me out. 
[License Plate Reader 3.3.4.txt](https://github.com/user-attachments/files/18132634/License.Plate.Reader.3.3.4.txt)

**Reply by jpetrone84** [reply]
@ChrisMaunder sorry not trying to be a PITA. But have you had a chance to take a look at the log?

**Reply by ChrisMaunder** [reply]
Sorry mate - it's just me at the moment and I'm flat-out.

**Reply by jpetrone84** [reply]
Ok, odd discovery. I updated my processor and now I can get a few of the modules to run. The license plate module will start but when I try to use explore to test it, it just results with not finding any plates on the test images. Also the object module isn't working I still get that error message while trying to use the coral accelerator. But it definitely seems like upgrading the CPU to a newer but still old processor helped. I didn't think the modules were CPU dependent to the point they wouldn't start or start and then stop.

**Comment by ChrisMaunder** [comment]
The ALPR module has a dependency on an object detection module that can detect license plates. We really need to add a check to ensure that's in place. The Object detection module using Coral doesn't provide plate detection. You will need to use one of the other Object Detection modules.

As a general guide, I always suggest NOT using Coral with Windows. It's just too flaky and unreliable.

Interesting about the CPU. That's a new one!
